src/components/ui/switch.jsx

```
@@ -1,19 +1,24 @@
   return (
     <button
       role="switch"
       aria-checked={checked}
       onClick={() => onCheckedChange(!checked)}
-      className={`relative inline-flex h-6 w-11 items-center rounded-full transition ${checked ? 'bg-blue-500' : 'bg-gray-300'} ${className}`}
+      className={clsx(
+        'relative inline-flex h-6 w-11 items-center rounded-full transition',
+        checked ? 'bg-cornflower-400' : 'bg-gray-300',
+        className
+      )}
     >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)